## ------------------ goal definition ------------------ ##

int8[] valid_colors
# Object color code
# 0 for ORANGE colored objects 
# 1 for YELLOW colored objects 
# 2 for BLUE colored objects 
# 3 for GREEN colored objects 
# 4 for RED colored objects 
# 5 for BLACK colored objects 
# 6 for PINK colored objects 
# 7 for PURPLE colored objects

bool verify_markers
# verify for markers over puck

int8[] valid_number_of_markers

---

## ----------------- result definition ----------------- ##

int8 color
# Object color code
# 0 for ORANGE colored objects 
# 1 for YELLOW colored objects 
# 2 for BLUE colored objects 
# 3 for GREEN colored objects 
# 4 for RED colored objects 
# 5 for BLACK colored objects 
# 6 for PINK colored objects 
# 7 for PURPLE colored objects

int8 number_of_markers
# number of marker over 

bool goal_achieved 		
# it is true if and only if the goal has been achieved

string message			
# message about rhe result


---

## ---------------- feedback definition ---------------- ##

float32 percentage		
# in range of 0% to 100%

string state			
# a string that describes the current state
